### DISCUSSION TITLE
Screen blank

### DISCUSSION THREAD

**Discussion by Barugon** [pop-os/cosmic-epoch#879] [Ideas]
Cosmic needs a configurable timeout and screen blank with the option to lock the screen. As it is, even if I manually lock the screen, it does not go blank.

There should also be a setting for completely turning off the display.

**Comment by throat-chop** [comment]
A built-in Caffeine would also be nice

**Reply by nalladev** [reply]
> A built-in Caffeine would also be nice

https://github.com/pop-os/cosmic-applets/issues/757

**Comment by git-f0x** [comment]
Display power saving options are planned, and may be implemented relatively soon: pop-os/cosmic-comp#517
